### ISSUE TITLE
[Includes suggested fix]: Ediff highlights rather unreadable

### ISSUE THREAD

**Issue by ianyepan** [nashamri/spacemacs-theme#162]
Here is a screenshot that showcases the highlights in ediff. Sadly, the high contrast of colours appear to be rather unreadable.

<img width="1280" alt="Screenshot 2020-03-22 at 22 26 28" src="https://user-images.githubusercontent.com/36686900/77252033-bff5ce00-6c8c-11ea-9b2e-6f344b148e5e.png">

### My fix:

I've been experimenting with the colours and decide to use the ones from **magit** diffs:

```emacs-lisp
(ediff-fine-diff-A        ((t (:background "#663333" :foreground "#eecccc" :extend t))))
(ediff-fine-diff-B        ((t (:background "#336633" :foreground "#cceecc" :extend t))))
(ediff-fine-diff-C        ((t (:background "#333366" :foreground "#ccccee" :extend t))))
```
Here is the result.

<img width="1280" alt="Screenshot 2020-03-22 at 22 25 34" src="https://user-images.githubusercontent.com/36686900/77252058-ed427c00-6c8c-11ea-93cb-d309fb6ce435.png">

IMHO, this looks a lot more consistent with the design principle of the rest of the colourscheme, and also improves readability of diff hunks.

**Comment by nashamri** [comment]
Your fix does look much better, I just need to find the time and prepare the alternative set of colors for the light variant. :+1:

**Comment by ianpan870102** [comment]
Absolutely! P.s. Magit does have some good colour choices that might of help :) 
Thx again for considering the change!
